### ISSUE TITLE
Disconnecting after sleep causes a hard crash.

### ISSUE THREAD

**Issue by TheQuinbox** [NVDARemote/NVDARemote#290]
If I leave my NVDA Remote connected when I put my computer to sleep, then wake it up and try to disconnect, my NVDA hard crashes, to the point that I just have to control Alt N to get it back.

**Comment by Simon818** [comment]
Have you tested latest remote? I haven't experienced this in a while and I know there were some other connection fixes in recent versions, but those weren't released as an official package until 2.5.

**Comment by TheQuinbox** [comment]
Yeah, still happens on Remote 2.5, NVDA 2022.1

**Comment by cary-rowen** [comment]
Yes, the issue can still be reproduced even with the latest 'master' branch build of the add-on.
Also, in addition to resuming from sleep, turning off the network and waiting a moment for the network to reconnect can also reproduce this.

cc @tspivey

**Comment by jcsteh** [comment]
I've seen this from time to time. I thought I'd found a reliable way to reproduce it, but... nope. Now I can't reproduce it at all no matter how many times I reconnect, disconnect, drop the network connection, etc.

Can anyone reproduce this 100% reliably? If so, can you provide steps?

**Comment by cary-rowen** [comment]
Happily, I can't reproduce this with master builds anymore either.

**Comment by TheQuinbox** [comment]
I haven't seen this with latest master, either.

**Comment by cary-rowen** [comment]
This Issue seems to be closed.

**Comment by ultrasound1372** [comment]
I can reproduce it with some build identifying itself as 2.6 that I got via add-on updater from one of the sources, but I'm unsure if it's truly the latest. NVDA 2023.1, can reliably reproduce by toggling airplane mode, waiting for the connection to come back, and then using the disconnect gesture.

**Comment by jcsteh** [comment]
I just fixed this in #324.

**Comment by ctoth** [comment]
Fix merged and released as v2.6.2
